tests/suite/text/raw.typ

```diff
@@ -648,6 +648,22 @@ fn main() {
 #let hi = "Hello World"
 ```
 
+--- raw-theme-set-to-none ---
+#set raw(theme: none)
+```typ
+#let foo = "bar"
+```
+
+#set raw(theme: auto)
+```typ
+#let foo = "bar"
+```
+
+#set raw(theme: none)
+```typ
+#let foo = "bar"
+```
```

Comment:
Since auto is already tested above, it suffices to have just the first of these three here, in my opinion.